### PR TITLE
Write style guide.

### PR DESCRIPTION
**Pull request by steveperry-53**
Request for comments on new style guide.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1619)

### CODE REVIEW

File: _data/support.yml [hunk 1]
```diff
@@ -14,6 +14,8 @@ toc:
     path: /docs/contribute/stage-documentation-changes/
   - title: Using Page Templates
     path: /docs/contribute/page-templates/
+  - title: Writing Style Guide
```
Comment (jaredbhatti) [inline _data/support.yml]: I would call this the "Documentation Style Guide".

**Comment by jaredbhatti** [comment]
Made a bunch more suggested changes.  Moved some stuff around again.  Phillip or Devin - any comments?

File: docs/contribute/style-guide.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+This page gives writing style guidelines for the Kubernetes documentation.  These are guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+For additional information on creating new content for Kubernetes, follow the instructions on [using page template](/docs/contribute/page-templates/) and creating a [creating a documentation pull request](/docs/contribute/create-pull-request/).
+{% endcapture %}
+
+{% capture body %}
+
+## Documentation formatting standards
+
+The following sections detail how to format API objects, code snippets,
```
Comment (devin-donnelly) [inline docs/contribute/style-guide.md]: Incomplete sentence?

I'd also make it clearer: "The following sections detail how to refer to Kubernetes API objects outside of code snippets, e.g. in paragraphs."

File: docs/contribute/style-guide.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    The output is similar to this:
+
+        NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+        nginx    1/1       Running   0          13s    10.200.0.4   worker0
+
+## Kubernetes.io word list
+
+A list of Kubernetes-specific terms and words to be used consistently across the site. 
+<table>
+  <tr><th>Term</th><th>Useage</th></tr>
```
Comment (devin-donnelly) [inline docs/contribute/style-guide.md]: Useage -> Usage

File: docs/contribute/style-guide.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+
+{% capture whatsnext %}
+* Learn about [writing a new topic](/docs/contribute/write-new-topic/).
+* Learn about [using page templates](/docs/contribute/page-templates/).
+* Learn about [staging your changes](/docs/contribute/stage-documentation-changes/)
+* Learn about [creating a pull request](/docs/contribute/create-pull-request/).
+{% endcapture %}
+
+{% include templates/concept.md %}
```
Comment (devin-donnelly) [inline docs/contribute/style-guide.md]: I might also suggest as an anti-pattern:

"Avoid referring to tasks as "simple", "easy", "complex", etc. Such modifiers are unnecessary and often subjective."

**Comment by devin-donnelly** [comment]
@steveperry-53 @jaredbhatti Added my comments.

File: _data/support.yml (hunk 1 above)
Comment (steveperry-53) [inline _data/support.yml]: Done.

File: docs/contribute/style-guide.md [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+## Documentation formatting standards
+
+The following sections detail how to format API objects, code snippets, 
+
+### Capitalize API objects
+
+Capitalize the names of API objects. Refer to API objects without saying
+"object."
+
+<table>
```
Comment (pwittrock) [inline docs/contribute/style-guide.md]: Why not use a MD table instead of html?

File: docs/contribute/style-guide.md [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>Set the value of the <code>replicas</code> field in the configuration file.</td><td>Set the value of the "replicas" field in the configuration file.</td></tr>
+  <tr><td>The <code>kubectl run</code> command creates a Deployment.</td><td>The "kubectl run" command creates a Deployment.</td></tr>
+</table> 
+
+### Don't include the command prompt.
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>kubectl get pods</td><td>$ kubectl get pods</td></tr>
```
Comment (pwittrock) [inline docs/contribute/style-guide.md]: Nevermind, I see this is addressed next

File: docs/contribute/style-guide.md [hunk 7]
```diff
@@ -0,0 +1,182 @@
...
+
+### Avoid using "we"
+
+Using "we" in a sentence can get very confusing, because the reader often doesn't know if they're part of the "we" you're describing. In most instances, you want to avoid using "we". 
+
+<table>
+  <tr><th>Do</th><th>Don't</th></tr>
+  <tr><td>Version 1.4 includes ...</td><td>In version 1.4, we have added ...</td></tr>
+  <tr><td>Kubernetes provides a new feature for ...</td><td>We provide a new feature ...</td></tr>
+  <tr><td>In this page, we are going to learn about pods.</td><td>This page will teach you how to use pods.</td></tr>
```
Comment (pwittrock) [inline docs/contribute/style-guide.md]: Are these swapped?  The last row says to use 'we'

File: docs/contribute/style-guide.md (hunk 5 above)
Comment (steveperry-53) [inline docs/contribute/style-guide.md]: I've come to fear Markdown tables. We can revisit for another PR if you like.

File: docs/contribute/style-guide.md (hunk 7 above)
Comment (steveperry-53) [inline docs/contribute/style-guide.md]: Fixed.

**Pull request merged**